### PR TITLE
Rename non-debug to non-release

### PR DESCRIPTION
**Pull request by letmaik**
Aligning terminology to README.

### CODE REVIEW

**Review by prp — APPROVED** [review]
LGTM

File: .azure-pipelines/scripts/build.sh [hunk 1]
```diff
@@ -15,8 +15,8 @@ fi
 # Initialize the variables.
 if [[ "$build_mode" == "debug" ]]; then
     make_args="DEBUG=true"
-elif [[ "$build_mode" == "nondebug" ]]; then
-    make_args="DEBUG=false"
+elif [[ "$build_mode" == "nonrelease" ]]; then
```
Comment (hukoyu) [inline .azure-pipelines/scripts/build.sh:18]: nondebug means release right? Shouldn't this be release?

File: .azure-pipelines/scripts/create_standalone_deb_pkg.sh [hunk 2]
```diff
@@ -47,7 +47,7 @@ if [ -z $SGXLKL_PREFIX ]; then
 fi
 
 if [ -z $SGXLKL_BUILD_MODE ]; then
-    echo "ERROR: 'SGXLKL_BUILD_MODE' is undefined. Please export SGXLKL_BUILD_MODE=<debug|nondebug>"
+    echo "ERROR: 'SGXLKL_BUILD_MODE' is undefined. Please export SGXLKL_BUILD_MODE=<debug|nonrelease>"
```
Comment (hukoyu) [inline .azure-pipelines/scripts/create_standalone_deb_pkg.sh:50]: Shouldn't this be debug|release?

File: .azure-pipelines/template.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ parameters:
   # Names must not have dashes as job names cannot have them.
   # NOTE: When changing this list, also change install_prefix_mapping.
   - debug
-  - nondebug
+  - nonrelease
```
Comment (hukoyu) [inline .azure-pipelines/template.yml:12]: release?

File: .azure-pipelines/template.yml [hunk 4]
```diff
@@ -32,7 +32,7 @@ parameters:
   type: object
   default:
     debug: /opt/sgx-lkl-debug
-    nondebug: /opt/sgx-lkl-nondebug
+    nonrelease: /opt/sgx-lkl-nonrelease
```
Comment (hukoyu) [inline .azure-pipelines/template.yml:35]: release?

File: tests/common.mk [hunk 5]
```diff
@@ -19,8 +19,8 @@ else
 	ifeq (${SGXLKL_BUILD_MODE},)
 		ifeq (${SGXLKL_PREFIX},/opt/sgx-lkl-debug)
 			export SGXLKL_BUILD_MODE=debug
-		else ifeq (${SGXLKL_PREFIX},/opt/sgx-lkl-nondebug)
-			export SGXLKL_BUILD_MODE=nondebug
+		else ifeq (${SGXLKL_PREFIX},/opt/sgx-lkl-nonrelease)
```
Comment (hukoyu) [inline tests/common.mk:22]: release?

File: tests/ltp/run_ltp_test.sh [hunk 6]
```diff
@@ -32,7 +32,7 @@ csv_filename="sgxlkl_oe_ltp_test_result_$(date +%d%m%y_%H%M%S).csv"
 echo "SI No, Test Name, Stdout logfile name, Stderr logfile name, Execution Status" > $csv_filename
 
 report_dir="report"
-# Delete all files except ltp-batch(debug).* or ltp-batch(nondebug).*
+# Delete all files except ltp-batch(debug).* or ltp-batch(nonrelease).*
```
Comment (hukoyu) [inline tests/ltp/run_ltp_test.sh:35]: release?

File: tools/sgx-lkl-docker [hunk 7]
```diff
@@ -28,7 +28,7 @@ function usage() {
     echo "    --name=<name>                                    Name of Docker image to be created."
     echo "    --host-cfg=<path>                                Path to host configuration file."
     echo "    --app-cfg=<path>                                 Path to app configuration file."
-    echo "    --mode=<debug|nondebug>                          Select the deployment mode of SGX-LKL (default: debug)"
+    echo "    --mode=<debug|nonrelease>                          Select the deployment mode of SGX-LKL (default: debug)"
```
Comment (hukoyu) [inline tools/sgx-lkl-docker:31]: release?

**Review by hukoyu — CHANGES_REQUESTED** [review]
nondebug means release. I didn't understand why nonrelease is used instad of nondebug. It should be release. Am I missing something here? @letmaik

**Comment by prp** [comment]
> nondebug means release. I didn't understand why nonrelease is used instad of nondebug. It should be release. Am I missing something here? @letmaik

@hukoyu no, see https://github.com/lsds/sgx-lkl/issues/144

**Review by hukoyu — APPROVED** [review]
After reading issue https://github.com/lsds/sgx-lkl/issues/144 Peter shared now it make sense.

**Pull request merged**